s3-proxy/s3.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/src/core/ngx_string.c b/src/core/ngx_string.c
+index 468e9600..7d8cfc9c 100644
```

Comment:
These first two lines shouldn't be here?

Reply:
Or do we need `git patch` if we retain these lines? Sorry, I've never seen this kind of patch, usually just `diff - u` output so not familiar.

Reply:
Oh, that's just what `git diff` outputs; it's harmless.

But, didn't know about `git apply` - I don't need to install `patch`.

Reply:
Oh wait, no, we don't have `git` here.